package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "audius-protocol",
+  "description": "Audius is working to create a fully decentralized community of artists, developers, and listeners collaborating to share and defend the world’s music.",
+  "dependencies": {
```

Comment:
dev-dependencies?

Reply:
i didn't think this would matter since we're not going to be publishing npm modules or building node apps from this folder. Also currently, if we say `npm install` in our docs, it'll pick this up, but if you make it dev then people will need to specify the save-dev option. seemed safe to make it default?

Reply:
you right, this is fine

Reply:
eh, not sure. but doesn't need to block merging this, huge to get this in